core/tests/protractor/historyTab.js

```diff
@@ -97,70 +97,85 @@ describe('Exploration history', function() {
         highlighted: true
       },
       10: {
-        text: '      feedback: []',
+        text: '      feedback:',
         highlighted: true
       },
       11: {
+        text: '        audio_translations: {}',
+        highlighted: false
+      },
+      12: {
+        text: '        html: \'\''
```

Comment:
Add a comma at the end of this line. Then add `highlighted: true` on the line below it.